src/main/java/oop/automobiles/DumpTruck.java

```diff
@@ -0,0 +1,20 @@
+package oop.automobiles;
+
+/*
+ * A class which defines all Dump Trucks.
+ */
+public final class DumpTruck extends Truck {
+
+    // Constructor method for DumpTruck
+    public DumpTruck(String license, int wheels) {
+        super(license, wheels);
+    }
+
+    public void loadGarbage(String item) {
+        this.getCargo().add(item);  // Adds items to the cargo list (loads garbage)
+    }
+
+    public void dumpGarbage(String item) {
```

Comment:
much better method, but once again there's no need for the String item argument -- it should just be dumpGarbage(), that way you don't have to pass null in the test which is generally a bad practice.